src/problem.cpp

```diff
@@ -108,7 +108,8 @@ void Problem::initialize(moveit::core::RobotModelConstPtr robot_model, const mov
             {
                 if(fixed_joint_name == joint_name)
                 {
-                    return (ssize_t)-1 - (ssize_t)robot_model->getVariableIndex(name);
+                    return static_cast<ssize_t>(-1) -
```

Comment:
Is that first static cast really necessary? Maybe it is, but usually the compiler is good about doing implicit conversions from literals without emitting warnings presuming the implicit conversion is known to be lossless for that given literal value which is the case for `-1` and `ssize_t`.